components/Accordion/Accordion.vue

```diff
@@ -79,5 +86,10 @@ const iconColor = computed(() => {
       return 'text-neutral-500'
   }
 })
-onUnmounted(() => unregister(accordionId))
+onMounted(() => {
+  if (route.hash === `#${titleAccordionId}`) {
+    open(titleAccordionId)
+  }
+})
+onUnmounted(() => unregister(titleAccordionId))
```

Comment:
Why this is required?

Reply:
This opens the corresponding accordion, otherwise it is closed